step-templates/upgrade-octopus-server.json

```diff
@@ -36,13 +33,26 @@
         "Octopus.ControlType": "SingleLineText"
       },
       "Links": {}
+    },
+    {
+      "Id": "098375bf-ff27-47c3-93ce-0191aade1b21",
+      "Name": "SpecificVersionInstallation",
+      "Label": "Version number of Octopus Deploy which is going to be installed",
+      "HelpText": "Selects a specific version of Octopus Deploy to install. This is especially handy for a downgrade.",
+      "DefaultValue": "#{Octopus.Release.Number}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
     }
   ],
-  "LastModifiedBy": "McAndersDK",
+  "Category": "octopus"
```

Comment:
@verzada apologies for the delay in getting to this!

The build failed as there's a `,` missing at the end of this line as this line has moved from the end of the JSON document. Which also means there's a `,` after the `$Meta` property that shouldn't be there.

I will review the script changes in a bit when I'm in front of my computer (instead of my mobile 🙂)

Reply:
Hey @alfhenrik 
I'm wondering if there's anything else lacking in the commit, since the commit hasn't passed through.